lib/akamai_rspec/matchers/non_akamai.rb

```diff
@@ -54,12 +54,18 @@ def ssl_client_for_verify_cert(tcp_client, addr, url)
     return false if URI(url).scheme == "http"
     url = "https://#{url}" unless URI(url).scheme
     begin
-      RestClient::Request.execute(method: :get, url: url, verify_ssl: verify)
+      # Avoid AkamaiRspec::Request as it turns off SSL checking
+      @response = RestClient::Request.execute(method: :get, url: url, verify_ssl: verify)
```

Comment:
`VERIFY_NONE` (connects via TLS but doesn't check certs)
`VERIFY_PEER` (validate certs )
`VERIFY_CLIENT_ONCE` (the openSSL docs are unclear on what this means)
`VERIFY_FAIL_IF_NO_PEER_CERT` (the openSSL docs are unclear on what this means)